man/ostree-pull.xml

```diff
@@ -103,7 +103,12 @@ Boston, MA 02111-1307, USA.
                 <term><option>--mirror</option></term>
 
                 <listitem><para>
-                    Write refs suitable for a mirror.
+                    Write refs suitable for a mirror, i.e. refs are stored in the
+                    <literal>heads/</literal> directory rather than the
+                    <literal>remotes/</literal> directory. This makes the target repo
+                    suitable to be exported for other clients to pull from. If no specific
+                    refs are specified, all refs will be fetched (the remote must have a
+                    summary file present).
```

Comment:
this is much better. I had sat down to write my own version and this almost matched exactly! Only thing I would change is maybe a little more detail here:

`This makes the target repo suitable to be a server for other clients to pull from as an ostree remote.`